src/main/kotlin/app/revanced/patches/tiktok/misc/settings/patch/SettingsPatch.kt

```diff
@@ -8,104 +8,110 @@ import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.addInstructions
 import app.revanced.patcher.extensions.instruction
 import app.revanced.patcher.extensions.replaceInstruction
-import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint.Companion.resolve
 import app.revanced.patcher.patch.BytecodePatch
 import app.revanced.patcher.patch.PatchResult
 import app.revanced.patcher.patch.PatchResultError
 import app.revanced.patcher.patch.PatchResultSuccess
 import app.revanced.patcher.patch.annotations.DependsOn
 import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
 import app.revanced.patches.tiktok.misc.integrations.patch.IntegrationsPatch
 import app.revanced.patches.tiktok.misc.settings.annotations.SettingsCompatibility
-import app.revanced.patches.tiktok.misc.settings.fingerprints.AboutViewFingerprint
+import app.revanced.patches.tiktok.misc.settings.fingerprints.AboutPageFingerprint
 import app.revanced.patches.tiktok.misc.settings.fingerprints.AdPersonalizationActivityOnCreateFingerprint
 import app.revanced.patches.tiktok.misc.settings.fingerprints.SettingsOnViewCreatedFingerprint
 import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.builder.instruction.BuilderInstruction21c
 import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
 import org.jf.dexlib2.iface.instruction.ReferenceInstruction
+import org.jf.dexlib2.iface.instruction.WideLiteralInstruction
 import org.jf.dexlib2.iface.instruction.formats.Instruction35c
-import org.jf.dexlib2.iface.reference.StringReference
 import org.jf.dexlib2.iface.reference.TypeReference
 
 @Patch
 @DependsOn([IntegrationsPatch::class])
-@Name("settings")
+@Name("tiktok-settings")
 @Description("Adds ReVanced settings to TikTok.")
 @SettingsCompatibility
 @Version("0.0.1")
 class SettingsPatch : BytecodePatch(
     listOf(
+        AboutPageFingerprint,
         AdPersonalizationActivityOnCreateFingerprint,
         SettingsOnViewCreatedFingerprint,
     )
 ) {
     override fun execute(context: BytecodeContext): PatchResult {
-        SettingsOnViewCreatedFingerprint.result?.let {
-            AboutViewFingerprint.resolve(context, it.method, it.classDef)
-        }
-        // Patch Settings UI to add 'Revanced Settings'.
-        val targetIndexes = findOptionsOnClickIndex()
-        with(SettingsOnViewCreatedFingerprint.result!!.mutableMethod) {
-            for (index in targetIndexes) {
-                if (
-                    instruction(index).opcode != Opcode.NEW_INSTANCE ||
-                    instruction(index - 4).opcode != Opcode.MOVE_RESULT_OBJECT
-                )
+
+        AboutPageFingerprint.result?.let {
+            val startIndex = it.scanResult.patternScanResult!!.startIndex
+
+            with(it.mutableMethod.implementation!!.instructions) {
+                copyrightPolicyLabelId = (this[startIndex] as WideLiteralInstruction).wideLiteral
+            }
```

Comment:
This is not required. You can do:

```suggestion
            copyrightPolicyLabelId = (it.mutableMethod.instruction(startIndex) as WideLiteralInstruction).wideLiteral
```